crates/forge-runner/src/printing.rs

```diff
@@ -61,7 +61,15 @@ fn format_detailed_resources(
     used_resources: &UsedResources,
     tracked_resource: ForgeTrackedResource,
 ) -> String {
-    let sorted_syscalls = sort_by_value(&used_resources.syscall_counter);
+    // Sort syscalls by call count
+    let mut sorted_syscalls: Vec<_> = used_resources
+        .syscall_usage
+        .clone()
+        .into_iter()
+        .map(|(a, b)| (a, b.call_count))
```

Comment:
There variables have different types (`DeprecatedSyscallSelector` and `SyscallUsage`), so using `a` and `b` is not adequate (unlike in the comparison below, where it fits perfectly):

```suggestion
        .map(|(selector, usage)| (selector, usage.call_count))
```